README.md

```diff
@@ -143,14 +149,15 @@ pip install -U pip
 pip install lxml‑4.5.0‑cp38‑cp38‑win_amd64.whl
 
 # Install label studio
-pip install label-studio
+pip install poetry
+poetry install label-studio
```

Comment:
does this need to change? looks like the original instructions called for installing LS from pypi, which you can totally still do with `pip`